pyshtools/meson.build

```diff
@@ -0,0 +1,51 @@
+python_sources = [
+  '__init__.py',
+  '_version.py',
+]
+
+pymod = import('python')
+python = pymod.find_installation(pure: false, modules: ['numpy'])
```

Comment:
The `modules` argument is actually a little dangerous, for at least two reasons:
- the check is done by importing the module, so it doesn't work if there's no importable Python package or if the import name doesn't match the package name
- it may interfere with cross-compiling

I'll follow up on that upstream, but I think it's best left out here.